resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,258 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+import "./device-verification"
+
+ApplicationWindow{
+	property var profile
+
+	id: userProfileDialog
+	height: 650
+	width: 420
+	modality: Qt.WindowModal
+	Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+	palette: colors
+
+	Connections{
+		target: deviceVerificationList
+		onUpdateProfile: {
+			profile.fetchDeviceList(profile.userid)
```

Comment:
Shouldn't the deviceVerificationList handle that by itself?

Reply:
Yeah this is for when making sure one the verification is done, the userprofile updates immediately and show the verified device, deviceVerificationList only maintains the list of ongoing flow.